flutter/test/navigation/time_to_display_tracker_test.dart

```diff
@@ -0,0 +1,173 @@
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: inference_failure_on_instance_creation
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter/src/navigation/time_to_display_tracker.dart';
+import 'package:sentry/src/sentry_tracer.dart';
+import 'package:sentry_flutter/src/navigation/time_to_initial_display_tracker.dart';
+
+import '../fake_frame_callback_handler.dart';
+import '../mocks.dart';
+
+void main() {
+  late Fixture fixture;
+
+  setUp(() {
+    TestWidgetsFlutterBinding.ensureInitialized();
+    fixture = Fixture();
+  });
+
+  tearDown(() {
+    fixture.ttidTracker.clear();
+  });
+
+  group('time to initial display', () {
+    group('in root screen app start route', () {
+      test('matches startTimestamp of transaction', () async {
+        final sut = fixture.getSut();
+
+        await sut.trackRegularRouteTTD(fixture.getTransaction(name: '/'),
+            startTimestamp: fixture.startTimestamp);
+
+        final transaction = fixture.hub.getSpan() as SentryTracer;
```

Comment:
```suggestion
        final transaction = fixture.getTransaction(name: '/') as SentryTracer;
        await sut.trackRegularRouteTTD(transaction, startTimestamp: fixture.startTimestamp);
```
this way it's easier to understand what starts a transaction and what finishes it
apply to other tests as well